wrappers/services.go

```diff
@@ -75,30 +76,109 @@ func generateSnapServiceFile(app *snap.AppInfo, opts *AddSnapServicesOptions) ([
 	return genServiceFile(app, opts)
 }
 
+// max returns the maximum of two integers. Why is this
+// not provided by golang?
+func max(a, b int) int {
+	if a < b {
+		return b
+	}
+	return a
+}
+
+func min(a, b int) int {
+	if b < a {
+		return b
+	}
+	return a
+}
+
+func formatCpuGroupSlice(grp *quota.Group) string {
+	if grp.CPULimit == nil {
+		return ""
+	}
```

Comment:
The comment in the header suggests that we always enable cpu accounting, but here if there is not limit we do not generate the CPUAccounting=true snippet?

Reply:
I have changed this in my latest commit, it now always add this, so now the comment should better line up! Thanks for pointing this out :+1: